pkg/controller.v1alpha3/suggestion/composer/composer.go

```diff
@@ -19,6 +20,12 @@ import (
 	"github.com/kubeflow/katib/pkg/controller.v1alpha3/util"
 )
 
+const (
+	defaultInitialDelaySeconds = 10
```

Comment:
Do we need higher value for defaultInitialDelaySeconds?  If defaultInitialDelaySeconds needed is just 10 sec, why did we see timeouts when we had set 60 sec to context timeouts earlier?

Reply:
Did we miss something earlier?

Reply:
SGTM, I will update it in #777